scripts/firedrake-install

```diff
@@ -604,6 +604,7 @@ else:
 if options.get("with_blas") == "download":
     # Download openblas if the user specifies.
     petsc_options.add("--download-openblas")
+    petsc_options.add("--download-openblas-make-options='USE_THREAD=0 USE_LOCKING=1 USE_OPENMP=0'")
     if osname == "Darwin":
         petsc_options.add("--CFLAGS=-Wno-implicit-function-declaration")
 elif options.get("with_blas") is not None:
```

Comment:
While we're here, can we flip the `openblas` packages on debian to the `seq` versions? That's in the docker container (just pull in my #1905).

Reply:
Merged #1905 into this branch (and closed that PR)